### DISCUSSION TITLE
How can we make Slash menu work for React

### DISCUSSION THREAD

**Discussion by umerjaved178** [ueberdosis/tiptap#2785] [General Questions]
I have seen that Slash Menu/command is in experimental phase but code is available for Vue
https://tiptap.dev/guide/menus#slash-commands
https://tiptap.dev/experiments/commands

Can someone help me how can I implement this in react?
Thanks :)

PS:
https://www.notion.so/help/guides/using-slash-commands

**Comment by dahei** [comment]
I found this React implementation: https://codesandbox.io/s/tiptap-react-slash-command-e3j3u
